doc/website/release-notes/iceoryx-unreleased.md

```diff
@@ -362,6 +363,25 @@
     #include "iceoryx_hoofs/cxx/algorithm.hpp"
     constexpr uint32_t MAX_VAL = algorithm::maxVal(3, 1890, 57);
     constexpr uint32_t MIN_VAL = algorithm::minVal(3, 1890, 57);
+
+20. `ReleativePointer::registerPtr` returns `cxx::optional`
+
+    ```cpp
+    // before
+    uint64_t id = RelativePointer::register(myPtr, 42);
+
+    if(id == INVALID_ID)
+    {
+        // Early exit
+    }
+
+    // after
+    auto maybeId = RelativePointer::register(myPtr, 42);
```

Comment:
```suggestion
    auto maybeId = RelativePointer::register(startAddress, numBytes);
```

We register a segment beginning at `startAddress` with size `numBytes` and get some id now associated with this segment (if successful).